mev_inspect/utils.py

```diff
@@ -1,5 +1,11 @@
+from enum import Enum
 from hexbytes._utils import hexstr_to_bytes
 
 
+class RPCType(Enum):
+    parity = 0
+    geth = 1
```

Comment:
Elsewhere we do like:
```
class RPCType(Enum):
    parity = "parity"
    geth = "geth"
```

which then allows you to in click etc do
```
str_rpc_type = "parity"
rpc_type = RPCType(str_rpc_type)
```
instead of custom conversion logic